docs/docs/dev/documentation/health.md

```diff
@@ -1,13 +1,25 @@
 # Health checks
 
-Health is an extremely important part of the microservice architecture, load balancers rely on the 
-health status of the service while picking endpoint from load balancer set. Also, users want the service state to be 
-observable through the GUI or CLI. Orchestrators like Kubernetes also want to know the service status to manage 
-lifecycle of the containers. 
-
-Kuma supports several aspects of the health checking. There are two policies which allows configuring active and passive
-health checks:
-[Health Check](../../policies/health-check) and [Circuit Breaker](../../policies/circuit-breaker).
+Health is an important aspect of a microservice architecture. Load balancers use the health status
```

Comment:
It might worth saying that these health tools will enable us to select endpoints to communicate to between DPPs. That's really what we're talking about. I think using the term "load balancer" here would just confuse users.